pilot/pkg/model/config.go

```diff
@@ -260,6 +260,14 @@ type ConfigStoreCache interface {
 
 	// HasSynced returns true after initial cache synchronization is complete
 	HasSynced() bool
+
```

Comment:
IMO, in 1.7 we should clean up a bit the interfaces, it's a big spaghetti of too generic abstractions and overhead. I.e. flatten the stores, remove the badly named 'cache', and harmonize the events based on the needs of config distribution.